### ISSUE TITLE
Compilation: QStorageInfo: No such file or directory

### ISSUE THREAD

**Issue by randoum** [onetransistor/winbootx#4]
Hi :)

```
# make
g++ -c -m64 -pipe -O2 -Wall -W -D_REENTRANT -fPIE -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -Iobj -Iobj -o obj/main.o src/main.cpp
In file included from src/mainwindow.h:39:0,
                 from src/main.cpp:22:
src/dostable.h:27:24: fatal error: QStorageInfo: No such file or directory
 #include <QStorageInfo>
                        ^
compilation terminated.
Makefile:383: recipe for target 'obj/main.o' failed
make: *** [obj/main.o] Error 1
```

```
# dpkg-query -S QStorageInfo
dpkg-query: no path found matching pattern *QStorageInfo*
```

And ```# find /usr/include/x86_64-linux-gnu/qt5 -iname QStorage*``` returns nothing

```
# uname -a
Linux 3.16.0-4-amd64 #1 SMP Debian 3.16.39-1 (2016-12-30) x86_64 GNU/Linux
```

Cheers

**Comment by onetransistor** [comment]
`QStorageInfo` is available since Qt 5.4. Probably you are using an older version of Qt.

**Comment by randoum** [comment]
Right, I got the fresh one from Debian depo. v 5.3.2
